<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_StatusA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_StatusB">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Thread A</string>
    </property>
    <widget class="QPushButton" name="btn_start_A">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pause_A">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resume_A">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume A</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Thread B</string>
    </property>
    <widget class="QPushButton" name="btn_start_B">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pause_B">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resume_B">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume B</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
